components/cognates/description.md

```diff
@@ -0,0 +1,35 @@
+# Cognates
+
+A major use case of wordlists in historical linguistics is for identifying and assembling
+cognate sets. Assigning forms to cognate sets is itself a (rather large) step
+in analyzing the wordlist data, but also serves as intermediate step before
+feeding the cognate sets into further analyses e.g. to determine language relatedness.
+Thus, being able to exchange data on cognate judgements related to wordlists
+is important, and covered by CLDF. 
+
+It is recommended that columns or other metadata describing the method used
+for the cognacy judgements and the alignments are added, but as yet no clear standard
+for these has evolved.
+
+** Partial Cognates **
+
+Like cognates, partial cognates refer to a form in a wordlist. But to make it
+possible to annotate parts of a form, the segmentation of the form
+**must** contain morpheme boundaries,
+i.e. the `FormTable` must contain a column with `propertyUrl`
+`http://cldf.clld.org/v1.0/terms.rdf#segments` where the secondary separator
```

Comment:
What is a secondary separator?
What is the primary separator in the example below?
Why is it not enough to index partial cognate judgements by segment indices, why must there be morpheme indices?
What about non-concatenative morphology, such as Umlaut? How would one even represent taht using morpheme boundaries?

Reply:
Good questions. However, the current effort is about a backwards compatible v1.1 of the spec. So I wouldn't want to touch any of the (possibly ill-defined) semantics of the current properties. (I may have wrapped the lines of this paragraph, making them show up in the diff although no text has been changed. My bad.)

That said, I agree that this particular specification may be debatable. But I wouldn't want to "fix it" in any ad-hoc way, but  rather admit that it was included prematurely. So the proper way to fix it would be
- publish datasets with a custom replacement of this property
- thereby showing the usefulness of such a replacement
- which can then be proposed for inclusion in the spec

That's basically the process the other additions for v1.1 went through, and I think it makes sense, particularly if the next version isn't another 4 year away.

Reply:
Hm. Having failed to read and internalize this bit before, I must admit that all my three current datasets are in this format – which is actually largely compatible, I guess: The thing that breaks down is the interpretation that my entries in the ‘segments’ list and down the line are morphemes. But given that splitting by separator is done immediately upon import and that the internal logic a piece of software may have to split each ‘morpheme’ into segments by spaces can only come into play afterwards, I think the behaviour in the end will be much the same, with some unintentional exploration of edge cases ;-)